doc/manual/rule_uid_schema.md

```diff
@@ -0,0 +1,139 @@
+<!---
+This Source Code Form is subject to the terms of the Mozilla
+Public License, v. 2.0. If a copy of the MPL was not distributed
+with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Rule UID Schema
+
+## Introduction
+
+The document describes the composition of the unique identifier (UID) to be associated with quality checker rules.
+
+The document describes composition of rule UIDs, declares some of the concepts to be used for ASAM rules, and provides an example for an UID for rule of the ASAM OpenDRIVE standard.
+
+## Rule UID Concepts
+
+The UID is a string which encapsulate a sequence of **concepts** that allow to identify immediately a rule across the different domain. The concept are ordered and separated via the separation character **`:`**.
+
+The concepts for the rule UID are:
+
+ * **Emanating Entity**: a domain name for the entity (organization or company) that declares the rule UID
+ * **Reference Standard**: a short string that represents the standard or the domain to which the rule is applied
+ * **Definition Settings**: the semantic version of the standard or the domain to which the rule appears or is applied **for the first time**
```

Comment:
```suggestion
 * **Definition Settings**: the version of the standard or the domain to which the rule appears or is applied **for the first time**
```

Is it always `semantic` version? IMHO, some standards (especially company internal ones) do not use semantic version. So I would simply remove the word.